.github/workflows/workflow_test.yml

```diff
@@ -138,6 +138,7 @@ jobs:
         galaxy-branch: ${{ env.GALAXY_BRANCH }}
         chunk: ${{ matrix.chunk }}
         chunk-count: ${{ needs.setup.outputs.chunk-count }}
+        planemo-version: 'https://github.com/galaxyproject/planemo/archive/refs/heads/master.zip'
```

Comment:
`master` is a moving target. Ideally this should be changed to pin a specific release before the PR is merged.